### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by ZewdieMc**
### Hey Reviewer, :wave:  
In this PR, I did the following [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md):

- [x] Create JavaScript array to store all information for all projects. :1234:  
- [x]  Create project cards dynamically from data stored in JavaScript Objects :100: 
- [x] Create popup (modal) for the `See Project` button in each card both for  :leafy_green: 
   - [x]  mobile :iphone: 
   - [x] desktop :computer:

### CODE REVIEW

**Review by KanzaTahreem — COMMENTED** [review]
# INVALID ❌ 

Hi @team 👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 😓

## Invalid Code Review Request ❌
- [ ] I think it would be nicer and more professional if you remove the placeholders associated with each header or customize it according to the project. For example `Describe 1 - 3 features you will add to the project.` should be removed/customized as you did for other headers/sections. Similarly, consider the same for **_Show your support_** heading These placeholders are there as a guide for you and not for readers 👩‍⚖️
- [ ] Also, please make sure to remove **template logo**. Optionally, you can replace with your own logo if youhave one.

_**Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready**_ 🤝🙏

Cheers and Happy coding!👏👨‍💻💻

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: css/modal.css [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  }
+
+  .description_wrapper p.description {
+    width: 90%;
+  }
+
+  .live-source-container a {
+    padding: 6px 6px;
+  }
+}
```
Comment (IngDavidTM) [inline css/modal.css]: - [ ] You did a fantastic job styling your popup:+1: :1st_place_medal:, but as a project requirement you should make sure that the popup is responsive, which means it should fit mobile and desktop sizes properly.
  - I've noticed that you are using an `overflow` to allow the scroll :+1:, but to match with the Figma you shouldn't use scroll inside the popup. 
  - Also kindly don't forget to align your popup to the middle of your page as is shown in Figma. You can check it below.

**Your design**

Screenshot from 2023-02-02 07-25-50

Screenshot from 2023-02-02 07-29-43

**Figma**

Screenshot from 2023-02-02 06-35-29

Screenshot from 2023-02-02 06-53-58

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @team 🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/modal.css (hunk 1 above)
Comment (ZewdieMc) [reply css/modal.css]: Hello @IngDavidTM , 
Thank you for the feedback. :pray:  
I am grateful. 

One thing I am not clear about is that when the scroll bar is removed from the popup, some contents are hidden. 
I believe the contents should not be hidden and the screen height is not enough to contain them without the scroll bar in place. The height of the Figma design is 1163px,  higher than a laptop's or Desktop's height. I would really appreciate any direction to solve this issue. :pray:

Comment (IngDavidTM) [reply css/modal.css]: Hi @ZewdieMc, yes it will be hidden but you should adjust the height of it to fit the screen size 👍

Comment (IngDavidTM) [reply css/modal.css]: You can use [vh](https://www.w3schools.com/cssref/tryit.php?filename=trycss_unit_vh) to fix that issue

File: css/modal.css [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+.live-source-container a {
+  border: 1px solid #6070ff;
+  text-decoration: none;
+  border-radius: 8px;
+  display: flex;
+  flex-wrap: nowrap;
+  padding: 5px 8px;
+  align-items: center;
+  gap: 10px;
+}
```
Comment (OyePriscilla) [inline css/modal.css:87]: - [ ] You're almost there! kindly ensure that your modal contains all the children elements as suggested earlier. The two `links` are displayed outside the modal container for the **`mobile version`**. 👍 

## Your Design
png-transparent-nigerian-naira-currency-symbol-symbol-miscellaneous-angle-text

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @ZewdieMc 😃, 

You've really done a great work, I must confess!

There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-so-close-seth-meyers

## Highlights
Github Flow observed✔️
Linters Set up correctly ✔️ 
Descriptive PR summary and title ✔️ 

## Required Changes ♻️

## Optional suggestions

- [optional] Please make sure to remove the Microverse template logo in the README file or you replace it with yours if you have one.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @ZewdieMc, 👋 

Your project is complete! Great job for following the project requirement🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 



## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

### Changes Required ♻️

- [x]  _Please check the comments under the review._

### Optional Changes 🔥

Thank You.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**